client/src/AddNewVideo.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+
+const AddNewVideo = (props) => {
+  const [title, setTitle] = useState("");
+  const [url, setUrl] = useState("");
+
+  const handleOnSubmit = (event) => {
+    event.preventDefault();
+    //Increasing of each video ID
+    const newId = (id) => {
+      return id++;
```

Comment:
I am not sure what are you trying to do here. If you are trying to get a new id number that is 1 bigger than the old one, you can just say `return id + 1;`. This way, it will be a more clear way.